### PR TITLE
static_website_tls: use logging instead of printf

### PR DESCRIPTION
**Pull request by azul**

### CODE REVIEW

File: static_website_tls/dispatch.ml [hunk 1]
```diff
@@ -9,9 +9,11 @@ module type HTTP = sig
   val listen: t -> IO.conn -> unit Lwt.t
 end
 
-module Dispatch (C: CONSOLE) (FS: KV_RO) (S: HTTP) = struct
+(* Logging *)
+let server_src = Logs.Src.create "server" ~doc:"HTTPS server"
+module Server_log = (val Logs.src_log server_src : Logs.LOG)
 
-  let log c fmt = Printf.ksprintf (C.log c) fmt
+module Dispatch (C: CONSOLE) (FS: KV_RO) (S: HTTP) = struct
```
Comment (talex5) [inline static_website_tls/dispatch.ml]: Don't think we need `C:CONSOLE` here any longer...

Comment (azul) [inline static_website_tls/dispatch.ml]: Ahh... nice!

**Comment by azul** [comment]
@talex5 this is great! I can get rid of all the console references. :smiley: Now i see what the logging thing is for.

**Comment by samoht** [comment]
Thanks!

**Pull request merged**